S2S Integration Samples/README.md

```diff
@@ -35,4 +35,12 @@ S2SIntegration_ProcessPrintTask is used for
 - Receiving a webhook notification when the Task is created.
 - Getting a Print Job for the task and reading the job attributes.
 - Updating the job's "Color" attribute to "GrayScale".
-- Completing the Task.
\ No newline at end of file
+- Completing the Task.
+
+## Demos
+- Demo: [Create Printer]()
+- Demo: [Hold print job and implement task trigger]()
+
+## Resources
+Code sample to generate a [Certificate Signing Request](https://docs.microsoft.com/en-us/universal-print/hardware/universal-print-oem-certificate-signing-request) with the BouncyCastle C# library.
+Documentation to generate a [printer device token](https://docs.microsoft.com/en-us/universal-print/hardware/universal-print-oem-printer-registration#3-getting-printer-token)
```

Comment:
why do we need printer device token?
we should be having two links instead of this one:

Documentation to generate a [user token] (to register a printer)
Documentation to generate an [application token] (to create task trigger, change notifications, update printer etc etc.)

Reply:
I can delete the resource to create a printer token, but we already show the code to generate a user token and an application token.